tools/dynamodbConnector.py

```diff
@@ -12,63 +12,61 @@ class DynamoDBConnector:
     def __init__(self):
         """Initialise the connection, and create a logger instance."""
         self.logger = logging.getLogger(__name__)
-        self.dynamodb = boto3.resource(
+        self.dynamodb = boto3.client(
             'dynamodb',
-            region_name='eu-west-2',
-            endpoint_url='http://localhost:8765'
```

Comment:
i assume that these come from env variables now? have you updated the readme file to mention this?

Reply:
It's coming from the ~./aws/credentials file. In aws this file is auto-generated by a special set of permissions, but I should mention it in the readme for local runs 👍